Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+pipeline {
+  agent {
+    label 'sbt-slave'
+  }
+
+  environment {
+    // Ensure that build scripts recognise the environment they are running within
+    CI = ''
+    SCALA_VERSION = '2.10.6'
```

Comment:
Will add in a `jenkins` value for the setting.

`SCALA_VERSION` here refers to the version of Scala that SBT will run. Without this setting, we will run SBT with Scala 2.10.6 (regardless of this being set within the project build.sbt). I'd rather be explicit regarding the version of Scala that SBT uses - particularly on a build server.